app/main.py

```diff
@@ -2,4 +2,70 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
```

Comment:
If argument can be None you should show it in type hint
```suggestion
            coords: list[int] | None = None
```